packages/collector/test/tracing/frameworks/fastify/app.js

```diff
@@ -5,21 +5,92 @@
 
 'use strict';
 
-require('../../../../')();
+const semver = require('semver');
+const FASTIFY_VERSION = process.env.FASTIFY_VERSION || '1.14.6';
+const FASTIFY_REQUIRE = semver.major(FASTIFY_VERSION) === 1 ? 'fastify' : `fastify${semver.major(FASTIFY_VERSION)}`;
+
+const mock = require('mock-require');
 
+// NOTE: Link e.g. fastify2 to fastify or fastify3 to fastify
+if (FASTIFY_REQUIRE !== 'fastify') {
```

Comment:
We have to link fastify, because our instrumentation of fastify needs it.
It listens on fastify `onModuleLoad` and not **fastify2**.

I didn't want to change the fastify instrumentation code and listen for `process.env.FASTIFY_VERSION` to change `onModuleLoad`.

Reply:
That absolutely makes sense.

Reply:
I extend the comment to don't confuse others too :)